### PR TITLE
Leaderboard updates: Model meta + task and benchmark info

### PR DESCRIPTION
**Pull request by x-tabdeveloping**
Tasks:
 - [x] Add Benchmark description and citation
 - [x] Add model info to table
 - [x] Add model filtering based on model meta
 - [x] Add task information table
 - [x] Replace mean rank with Borda rank

### CODE REVIEW

**Comment by x-tabdeveloping** [comment]
@isaac-chung yes!!

```
pip install mteb
git clone https://github.com/embeddings-benchmark/mteb.git
cd mteb
git checkout leaderboard_2
```
Then run this:
```python
from mteb.leaderboard import demo

demo.launch()
```

File: mteb/leaderboard/table.py [hunk 1]
```diff
@@ -1,16 +1,42 @@
...
 
+from mteb.models.overview import get_model_meta
 from mteb.overview import get_task
 
 
+def borda_count(scores: pd.Series) -> pd.Series:
+    n = len(scores)
+    ranks = scores.rank(method="average", ascending=False)
+    counts = n - ranks
+    return counts
```
Comment (KennethEnevoldsen) [inline mteb/leaderboard/table.py:17]: Yep - The only problem now is ties, which we calculate slightly differently in the paper (but I don't believe there are practically any ties), so I think it is equivalent so let us keep it as is.

**Review by KennethEnevoldsen — APPROVED** [review]
Looks good!

File: mteb/leaderboard/table.py (hunk 1 above)
Comment (x-tabdeveloping) [reply mteb/leaderboard/table.py:17]: I think I checked and this piece of code is equivalent to tournament borda. Is that not what we used in the paper?

Comment (KennethEnevoldsen) [reply mteb/leaderboard/table.py:17]: I don't think it is equivalent in the case of ties (but there are no ties so equivalent in practice). We should just remove the tournament borda description from the paper.

**Pull request merged**